### ISSUE TITLE
[Question] Any method to obtain Bounding box of a block or DIM Block

### ISSUE THREAD

**Issue by alison84r** [haplokuon/netDxf#152]
Hello Team,
I have one question. Is there any way I can get directly bounding box of the BLOCKS, Dimensions or Its respective blocks. 
Or do I have to calculate? If yes what are the parameters to consider for calculation of bounding box.
![image](https://user-images.githubusercontent.com/12997350/71231000-fb8d9e80-2311-11ea-8922-d2b99a4362ac.png)
![image](https://user-images.githubusercontent.com/12997350/71231016-0ea06e80-2312-11ea-9b5a-b528b2236aa3.png)

Let me know if my question is not clear. 
Thanks in advance.

**Comment by haplokuon** [comment]
No. There are some entities I cannot calculate it, like Text, MText, and Tolerance; I have no way of measuring strings based in TTF and SHX fonts. Also the Underlay and Shape entities have its size defined by the scale of its original dimensions that comes from the PDF, DWF, DGN, and SHX file.

**Comment by LeiYangGH** [comment]
unfortunately!
